includes/Modules/Idea_Hub.php

```diff
@@ -347,12 +354,19 @@ private function admin_notice_idea_hub_ideas( $notices ) {
 			self::SLUG_NEW_IDEAS,
 			array(
 				'content'         => function() use ( $escape_and_wrap_notice_content ) {
-					$message = sprintf(
-						/* translators: %s: URL to new ideas */
-						__( 'Site Kit by Google: Want some inspiration for a new post? <a href="%s">Review your new ideas</a> in Site Kit.', 'google-site-kit' ),
+					$message_body = sprintf(
+						/* translators: %s: URL to saved ideas */
+						__( 'Want some inspiration for a new post? <a href="%s">Review your new ideas</a> in Site Kit.', 'google-site-kit' ),
 						esc_url( $this->context->admin_url( 'dashboard', array( 'idea-hub-tab' => 'new-ideas' ) ) )
 					);
 
+					$message = sprintf(
+						/* translators: 1: Plugin name. 2: Notice content. */
```

Comment:
```suggestion
						/* translators: 1: Plugin name. 2: Message. */
```

Reply:
Sincere apologies for missing these on. Thank you so much for addressing!